pages/how-to-contribute.md

```diff
@@ -19,9 +19,19 @@ This document provides basic instructions for contributing to Accumulo.  If you
 
 ## Issues
 
-Any contribution should have a corresponding issue. Accumulo uses [JIRA] for issue tracking. Before creating an issue,
-you will need to create an [Apache JIRA account][jira-signup]. If you need help finding an issue to work on, check out
-the [open issues labeled for newbies][newbie-issues] or [contact us][contact].
+Accumulo uses GitHub issues to track bugs and features.  Each git repository
```

Comment:
I understand phrasing this for the eventual state when we have everything transitioned off of JIRA, but we're not there yet. Add an exception for the accumulo.git repo for now?

Reply:
I am proposing that we stop using JIRA for new issues.

Reply:
My reading off the mailing list discussion gives me the impression that people are ok with the goal of transitioning as long as there is a starting plan and the eventual goal of not using JIRA.  Hopefully this proposal provides both, if not it needs to be improved.

Reply:
Right, for the time being we will need to query github and JIRA for open issues before releasing.  We will also need to query both when constructing the release notes.  This will need to happen until JIRA has no issues for a release.  I can document this.

We could speed this along by adding the following request for new issues.

If you start working on an issue that exists in JIRA, please do the following :
 * Open a new github issue that links to the JIRA issue.
 * Close the JIRA issue.

If we did this, then we should be done with JIRA after all branches have had a release after the start of using GH issues.

Reply:
Thanks. I don't think there is any "hard" problem that will prevent us from making this work. I just want to make sure we all understand how to do it. I appreciate the effort to put this in writing.

Reply:
I understood the mailing list discussion to mean that accumulo-{website,examples,etc} would use GH Issues, while accumulo [core] continues to use JIRA. Did I misread that?

Reply:
> I understood the mailing list discussion to mean that accumulo-{website,examples,etc} would use GH Issues, while accumulo [core] continues to use JIRA. Did I misread that?

Mike started discussion of that option because he felt there was lack of consensus on moving everything to github issues. My read on the cause of the lack of consensus was the lack of a concrete starting plan that had a firm goal of moving off JIRA.  So I am proposing concrete starting plan. 

Nothing has been voted on. Personally I would like to see a vote on an option that seems to have have consensus in discussion.  I am posting this proposal for discussion.

Reply:
> I understood the mailing list discussion to mean that accumulo-{website,examples,etc} would use GH Issues, while accumulo [core] continues to use JIRA. Did I misread that?

I said we hadn't reach consensus on a plan to move Accumulo core to GH issues. However, Keith is proposing a plan here. If there is consensus on this plan, we can move all repos to GH issues.